src/lib.rs

```diff
@@ -1178,6 +1189,37 @@ where
     }
 }
 
+/// A thread-unsafe version of [`set_logger`].
+///
+/// This function is available on all platforms, even those that do not have
+/// support for atomics that is needed by [`set_logger`].
+///
+/// In almost all cases, [`set_logger`] should be preferred.
+///
+/// # Safety
+///
+/// This function is only safe to call when no other logger initialization
```

Comment:
I think it's probably a good idea to explicitly mention that it is safe to concurrently log while this function runs.